delete/redis.go

```diff
@@ -0,0 +1,32 @@
+package delete
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	storage "github.com/ninech/apis/storage/v1alpha1"
+	"github.com/ninech/nctl/api"
+)
+
+type redisCmd struct {
+	Name        string        `arg:"" help:"Name of the Redis resource."`
+	Force       bool          `default:"false" help:"Do not ask for confirmation of deletion."`
+	Wait        bool          `default:"true" help:"Wait until Redis is fully deleted"`
```

Comment:
Nit: for consistency with the other help texts.

```suggestion
	Wait        bool          `default:"true" help:"Wait until Redis is fully deleted."`
```